.sync.yml

```diff
@@ -10,9 +10,12 @@ Gemfile:
   optional:
     ':development':
       - gem: 'puppet-resource_api'
-        git: 'https://github.com/michaeltlombardi/puppet-resource_api'
-        branch: 'gh-225/main/custom-insync'
```

Comment:
since the feature is only in the very latest version, it might make sense to force that version to document/ensure it's always there.